backend/inspirehep/sitemap/cli.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import os
+from itertools import islice
+
+import click
+from flask import current_app, render_template, url_for
+from flask.cli import with_appcontext
+
+from inspirehep.utils import get_inspirehep_url
+
+from .sitemap import generate_sitemap_items
+
+
+@click.group()
+def sitemap():
+    """Command group to handle file operations."""
+
+
+def get_sitemap_page_filename(page):
+    return f"sitemap{page}.xml"
+
+
+def get_sitemap_page_absolute_url(page):
+    base_url = get_inspirehep_url()
+    return f"{base_url}/{get_sitemap_page_filename(page)}"
+
+
+def write_sitemap_page_content(page, page_content):
+    path = current_app.config["SITEMAP_FILES_PATH"]  # FIXME: should we mkdir -p?
```

Comment:
I'd add if ``os.path.isdir`` and raise if it's not present, but not creating because on prod might hide issues

Reply:
`write` already raises if dir is not there.